packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -38,6 +38,13 @@ interface BaseConfiguration {
 	 * Rewrite rules
 	 */
 	rewriteRules?: RewriteRule[];
+
+	// TODO: Can we reduce coupling between the request handler and WP?
+	/**
+	 * Answers whether the given path is a known remote asset path
+	 * for a minified WP build.
+	 */
+	isKnownRemoteAssetPath?: (wpRelativePath: string) => boolean;
```

Comment:
This change is in a generic `php-wasm` package, let's use naming that doesn't talk about WordPress.